configs/quota/quota.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+package quota;
+
+package istio.mixer.config;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/descriptor.proto";
+
+
+// Modeled on QuotaDescriptor
+// All fields except 
+message Template{
+  // name is a required field for all templates.
```

Comment:
`name` field should be an annotation providing the value that defines the name of the Template. The name is what types would reference.. right ?

I tried to model this as https://github.com/guptasu/mixer/pull/6/files#diff-a0ad06bac1ec4482efcd556f66391bd6R10

May be there is a better way.